src/lsst/cmservice/models/queue.py

```diff
@@ -54,7 +54,7 @@ class QueueUpdate(QueueBase):
     # Options based to process
     options: dict | str | None = None
     # When this was added to Queue
-    time_created: datetime
+    time_created: datetime | None = None
```

Comment:
Would it be reasonable to just init to current time here, rather than allowing None?

Reply:
Oh, or maybe "time_created" just shouldn't be updateable?

Reply:
Sure.  But you probably want to be able to lie about your age at some point...